api/src/main/java/org/openmrs/module/labonfhir/api/scheduler/FetchTaskUpdates.java

```diff
@@ -119,13 +120,21 @@ public void execute() {
 				lastRequstDate = dateFormat.format(lastRequest.getRequestDate());
 			}
 			
+			String practitionerId = config.getLisUserUuid();
+
 			String currentTime = dateFormat.format(newDate);
 			DateRangeParam lastUpdated = new DateRangeParam().setLowerBound(lastRequstDate).setUpperBound(currentTime);
 			
 			// Get List of Tasks that belong to this instance and update them
 			Bundle taskBundle = client.search().forResource(Task.class)
 			        .where(Task.IDENTIFIER.hasSystemWithAnyCode(FhirConstants.OPENMRS_FHIR_EXT_TASK_IDENTIFIER))
-			        .where(Task.STATUS.exactly().code(TaskStatus.COMPLETED.toCode())).lastUpdated(lastUpdated)
+					.where(Task.OWNER.hasId(practitionerId))
+			        .where(Task.STATUS.exactly().codes(
+						TaskStatus.COMPLETED.toCode(), 
+						TaskStatus.REQUESTED.toCode(), 
```

Comment:
@azizdiakite can you remove the` REQUESTED` status since its the innitioal status?

Reply:
oh yeah sure

Reply:
Done